package.json

```diff
@@ -103,5 +103,39 @@
     "packages": [
       "packages/ckeditor5-dev/packages/ckeditor5-dev-tests"
     ]
+  },
+  "resolutions": {
+    "ckeditor5": "35.1.0",
```

Comment:
Is this necessary? There's no such package AFAIK?

Reply:
@Dumluregn during installation github-writer without marking `ckeditor5` in resolutions was causing an error with `duplicated modules`.